plugins/woocommerce/src/Internal/Email/OrderPriceFormatter.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * OrderPriceFormatter class file.
+ */
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Internal\Email;
+
+use WC_Abstract_Order;
+
+/**
+ * Helper class for formatting prices in order emails.
+ *
+ * @internal Just for internal use.
+ */
+class OrderPriceFormatter {
+
+	/**
+	 * Gets item subtotal - formatted for display in emails.
+	 *
+	 * @param WC_Abstract_Order $order Order instance.
+	 * @param object            $item Item to get unit price from.
+	 * @param string            $tax_display Incl or excl tax display mode.
+	 * @return string Formatted item subtotal.
+	 */
+	public static function get_formatted_item_subtotal( WC_Abstract_Order $order, $item, string $tax_display = '' ): string {
```

Comment:
Thanks for strictly typing the new class!
Based on the `woocommerce_order_item_meta_start` hook usages, it looks like `$item` can be typed as `\WC_Order_Item`.

Reply:
Good point, updated in https://github.com/woocommerce/woocommerce/pull/57573/commits/522db14793b77747b5e02bbf1aa67b4202bddf28.